testkit-backend/2cypher.go

```diff
@@ -201,7 +202,17 @@ func nativeToCypher(v any) map[string]any {
 				"z":      x.Z,
 			},
 		}
+	case *db.InputPosition:
+		if x != nil {
+			return valueResponse("CypherMap", map[string]any{
+				"column": nativeToCypher(int64(x.Column)),
+				"line":   nativeToCypher(int64(x.Line)),
+				"offset": nativeToCypher(int64(x.Offset)),
+			})
+		}
+		return nil
 	}
```

Comment:
This should never happen. The server cannot send this type and the driver shouldn't put this type into anything that's supposed to polyfill for things that could come from the server.

Reply:
See above conversation, this was happening due to passing InputPosition to diagnostic record rather than a raw map, this is now removed.